test/schematype.test.js

```diff
@@ -281,4 +281,29 @@ describe('schematype', function() {
       });
     });
   });
+  it('demonstrates the `validateAll()` function (gh-6910)', async function() {
```

Comment:
```suggestion
  it('demonstrates the `validateAll()` function (gh-6910)', function() {
```

function does not need to be async, from what i can tell